src/ImGui.NET/ImGuiNative.cs

```diff
@@ -195,23 +191,13 @@ public static unsafe class ImGuiNative
         [DllImport(cimguiLib)]
         public static extern void igDummy(Vector2* size);
         [DllImport(cimguiLib)]
-        public static extern void igIndent();
+        public static extern void igIndent(float indent_w);
```

Comment:
Should this have a default parameter? Or was this one just bogus before? (I don't think I've called this myself).